src/status_im/contexts/wallet/common/token_value/view.cljs

```diff
@@ -34,11 +34,13 @@
    :on-press            #(rf/dispatch [:open-modal :screen/wallet.share-address {:status :receive}])})
 
 (defn- action-bridge
-  []
+  [token-data]
   {:icon                :i/bridge
    :accessibility-label :bridge
    :label               (i18n/label :t/bridge)
-   :on-press            #(js/alert "to be implemented")})
+   :on-press            (fn []
+                          (rf/dispatch [:hide-bottom-sheet])
+                          (rf/dispatch [:wallet/bridge-select-token {:token token-data}]))})
```

Comment:
I think it should also use the wizar-navigate-forward method here too? - it should be used for each step in the navigation

Reply:
@J-Son89 `wallet/bridge-select-token` calls `wallet/wizard-navigate-forward`